master/buildbot/test/unit/test_contrib_buildbot_cvs_mail.py

```diff
@@ -118,9 +126,11 @@ class TestBuildbotCvsMail(unittest.TestCase):
         skip = ("'%s' does not exist (normal unless run from git)"
                 % buildbot_cvs_mail_path)
 
-    def assertOutputOk(self, xxx_todo_changeme4, regexList):
+    def assertOutputOk(self, result, regexList):
```

Comment:
You can't use (output, code) because tuple parameter unpacking is a syntax error in Python 3: https://www.python.org/dev/peps/pep-3113/